src/pages/hosts/HostsTable.tsx

```diff
@@ -141,9 +142,10 @@ export const HostsTable: React.FC<Props> = ({
         placeholder: "Search ID",
         value: hostIdValue,
         onChange: onChangeHostId,
-        dataCy: "host-id-filter",
-        updateUrlParam: updateHostIdUrlParam,
-        resetUrlParam: resetHostIdUrlParam,
+        "data-cy": "host-id-filter",
+        onFilter: updateHostIdUrlParam,
+        onReset: resetHostIdUrlParam,
+        submitButtonCopy,
```

Comment:
It doesn't look like `submitButtonCopy` is being used to override "Filter" 🤔 
<img width="305" alt="image" src="https://user-images.githubusercontent.com/9298431/133469252-5985f843-ac2c-4a57-bc20-64abcd5ad5b3.png">